### ISSUE TITLE
Video element without a track's  kind attribute set to captions

### ISSUE THREAD

**Issue by Rafael-Luna97** [Rafael-Luna97/github-integration#87]
**Rule description**  
One or more video elements, without an ARIA-assigned role, not intentionally hidden in the DOM and available to assistive technologies, does not have a track element with kind set to captions as a direct child element

**Tool name**  
Access Engine

**Severity**  
Critical

**WCAG Success criteria**  
[1.2.2 Captions (Prerecorded) - Level A](https://www.w3.org/WAI/WCAG22/Understanding/captions-prerecorded)  
[1.2.4 Captions (Live) - Level AA](https://www.w3.org/WAI/WCAG22/Understanding/captions-live)

**Why it matters**  
When synchronized captions are not provided for synchronized media, users who are deaf or hard of hearing may not have access to important audio including spoken content that is in the media.

**What to do**  
Provide synchronized captions for synchronized media. The method for providing captions varies based on video format and video player, however, use of the track element with kind attribute set to <code>captions</code> is a common approach that is recommended.  

Captions need to include:
<ul>
  <li>All dialog</li> 
  <li>Important non-dialog audio content including sounds and music which are necessary for understanding</li> 
  <li>Identification of the speaker, if more than one speaker or person is present</li>
 </ul>

Quality captions need to be: 
<ul> 
  <li>Accurate</li> 
  <li>Consistent</li> 
  <li>Clear</li>
  <li>Readable</li> 
  <li>Equal</li> 
</ul>

<p>For videos with a purely musical audio track, it is recommended to put a single [MUSIC PLAYING] caption frame that drops off-screen after 10 seconds or so. The same would be true of a completely silent file - using a [NO SPEECH] indicator or something similar. In order to create an equal experience for Deaf/Hard-of-Hearing viewers, it is important to articulate the auditory experience - even if there is no auditory experience (otherwise, they might think they're missing something).</p> 

<p><em>Note-</em> Captions need to appear in the same language as the audio or video they are an equivalent for.</p>

**Task in the Level Access Platform**  
https://tenanta.localhost-dashboard.essentialaccessibility.com/my-tasks/6a7a30cf410cee05c8b1b588/6a7ccc958e278fc618efd6da

**Comment by Rafael-Luna97** [comment]
Task pushed to GitHub Issues by Rafael Luna from Level Access Platform.

Posted by Rafael Luna via Level Access

**Comment by Rafael-Luna97** [comment]
Task pushed to GitHub Issues by Rafael Luna from Level Access Platform.

Posted by Rafael Luna via Level Access

**Comment by Rafael-Luna97** [comment]
<a href="https://tenanta.localhost-dashboard.essentialaccessibility.com/tasks/6a7ccc958e278fc618efd6da/attachments/6a7ccc958e278fc618efd6d8/download" target="_blank" rel="noopener noreferrer">Linked Findings.csv</a>

**Comment by Rafael-Luna97** [comment]
Task pushed to GitHub Issues by Rafael Luna from Level Access Platform.

Posted by Rafael Luna via Level Access

**Comment by Rafael-Luna97** [comment]
Task pushed to GitHub Issues by Rafael Luna from Level Access Platform.

Posted by Rafael Luna via Level Access
